Sources/NIOSSL/SSLContext.swift

```diff
@@ -44,7 +46,7 @@ internal enum FileSystemObject {
         }
 
 #if os(Android) && arch(arm)
-        return (statObj.st_mode & UInt32(Glibc.S_IFDIR)) != 0 ? .directory : .file
```

Comment:
This `Glibc` scoping on Android armv7 was added for whatever reason in #45, but my testing now shows it isn't needed, so I removed it here and below.